htdocs/accountancy/bookkeeping/card.php

```diff
@@ -121,7 +127,7 @@
 	$error = 0;
 
 	if ($cancel) {
-		header("Location: ".DOL_URL_ROOT.'/accountancy/bookkeeping/list.php');
+		header("Location: ".DOL_URL_ROOT. $backtopage . (!empty($type)?'?type=sub':''));
```

Comment:
The $backtopage should already contains the DOL_URL_ROOT so fix to 
... Location: ".$backtopage ...

Reply:
Thanks for your return @eldy . Already corrected !